gef.py

```diff
@@ -10614,7 +10614,6 @@ def invoke(self, args: str, from_tty: bool) -> None:
                 try:
                     setting = gef.config.raw_entry(key)
                 except Exception as e:
```

Comment:
Can you also remove the `as e` since it's not used?

Reply:
Okay on it